### ISSUE TITLE
Fail to converge wavernn on LJ-speech dataset.

### ISSUE THREAD

**Issue by Yeongtae** [m-toman/tacorn#5]
I'm testing it on v100 gcp machine using my pretrained tacotron checkpoint.
But it fail to generate intelligible voice.

Is there any idea about it?

https://www.dropbox.com/s/pl2hkiot6vtlljz/WaveRNN%20test.zip?dl=0
[816k.zip](https://github.com/m-toman/tacorn/files/2477598/816k.zip)
Here is the test result.

**Comment by m-toman** [comment]
Interesting, it obviously learns but babbling.
I've tried a couple different data sets meanwhile (although always starting with the LJ base model) and this never happened, just the quality gradually from noisy to rather clear.

So I suspect there is some misalignment between the spectrograms and the raw data you feed in.

Did you use GTA output of Tacotron or extracted the features from the wavs yourself?

**Comment by Yeongtae** [comment]
I've used GTA output from My own repository and model.
https://github.com/Yeongtae/Tacotron-2

Now, I saw some difference in hparam.py.
![image](https://user-images.githubusercontent.com/13597229/46937490-67380400-d09c-11e8-9f1b-e1153636b0df.png)
It's yours.

![image](https://user-images.githubusercontent.com/13597229/46937535-7dde5b00-d09c-11e8-91d4-2eb8a5dc9524.png)
It's mine.

In my guess, the difference of SR make some error.
